test/utilities/constants.ts

```diff
@@ -17,3 +17,13 @@ export const WORKSPACE_SETTING_KEYS = {
 export const LOG_LEVELS = ['trace', 'debug', 'info', 'warn', 'error', 'silent'] as const;
 
 export type LogLevel = (typeof LOG_LEVELS)[number];
+
+// the name of repos
+export enum repoKeywords {
+  DeployInv = 'DeployInv'
+}
+
+// a map from a repo name to git url
+export const projectMaps = new Map<repoKeywords, string>([
```

Comment:
So we are going to maintain such a map to make e2e tests download them from Github.

Reply:
We have been discussing the need to expose our testing framework  How do you see us supporting such a structure when customers are using our framework?

The current structure of test/workspace initialization is configuration based, while this solution makes it a code based.

I am in favor of tooling that simplifies the test authors productivity, so I look forward to alternative suggestions. 

Please keep in mind local development and CI/CD.